superset-frontend/src/explore/components/SaveModal.tsx

```diff
@@ -256,7 +312,8 @@ class SaveModal extends React.Component<SaveModalProps, SaveModalState> {
               checked={this.state.action === 'saveas'}
               onChange={() => this.changeAction('saveas')}
             >
-              {t('Save as...')}
+              {' '}
```

Comment:
Curious why is the extra whitespace needed here?  It can be accomplished in one line using a template string like this:
{\` ${t('Save as ...')} \`}
Template strings render all whitespace between the opening and closing `
Seems like the default styling on the Radio component should handle this for consistency?

Reply:
running lint-fix does this for some weird reason. other wise the linter will complain